src/commands/create-git-tag.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+# How to author commands: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands
+description: >
+  Create new git tag based on the version in the package.json
+
+parameters:
+  bypass:
+    description: Does not create new tag if set to true
+    default: false
+    type: boolean
+
+steps:
+  - when:
+      condition: <<parameters.bypass>>
+      steps:
+        - run:
+            name: Bypass git tag creation
+            command: |
+              echo "bypass parameter set to true. Bypassing git tag creation..."
+              exit 0
+  - when:
+      condition:
+        not: <<parameters.bypass>>
+      steps:
+        - run:
+            name: Setup git
+            command: |
+              git config user.email "mdonnalley@salesforce.com"
```

Comment:
These shouldn't be hardcoded to yours. We could allow them to be passed in but maybe set it to the team email and set the name to "Release Orb" or something like that.